docs/application/native/guides/multimedia/openal.md

```diff
@@ -44,6 +44,11 @@ The main features of the OpenAL API include:
 
   > **Note**  
   > The device cannot fall into a sleep state while the context is not destroyed. To avoid unwanted battery consumption, destroy the context punctually.
+  >
+  > Since 3.0, extension alcDevicePauseSOFT() was introduced which allows applications to pause a playback device explictly.
+  > The main purpose is to silence output, stop processing, and to allow audio hardware to go into low-power mode.
+  > Additionally, you can use alcDeviceResumeSOFT() function to resume playback of a paused device.
```

Comment:
Please change alcDeviceResumeSOFT()  to ' alcDeviceResumeSOFT() '

Reply:
done, applied to alcDevicePauseSOFT() as well as alcDeviceResumeSOFT()